designsafe/static/scripts/ng-designsafe/services/project-service.js

```diff
@@ -162,7 +162,7 @@ export function ProjectService(httpi, $interpolate, $q, $state, $uibModal, Loggi
         } else if (project.value.projectType === 'field_recon') {
             requiredSets = [
                 'planning_set',
-                'socialscience_set',
+                // 'socialscience_set',
```

Comment:
This line can probably be deleted, too.

Reply:
Ah this is okay, this whole section is being overwritten in another PR.